insights/client/insights_spec.py

```diff
@@ -44,7 +44,8 @@ def get_output(self):
         '''
         # all commands should timeout after a long interval so the client does not hang
         # prepend native nix 'timeout' implementation
-        timeout_command = 'timeout %s %s' % (self.config.cmd_timeout, self.command)
+        timeout_command = 'timeout -s KILL %s %s' % (
```

Comment:
Did you consider using `-k` instead of starting off with the `KILL`?

Reply:
I did but I thought that might affect our timeout interval.

Reply:
imho if we lover the command timeout, that shouldn't be a big concern.  we could even just do `-k 5` to give a small buffer for graceful termination.